packages/electron-webpack/src/targets/RendererTarget.ts

```diff
@@ -203,7 +203,9 @@ async function generateIndexFile(configurator: WebpackConfigurator, nodeModulePa
     html = html.replace("</head>", `<script>require('module').globalPaths.push("${nodeModulePath.replace(/\\/g, "/")}")</script></head>`)
   }
 
-  html = html.replace("</head>", '<script>require("source-map-support/source-map-support.js").install()</script></head>')
+  if (process.env.ELECTRON_WEBPACK_DISABLE_SOURCE_MAP_SUPPORT) {
```

Comment:
Shouldn't this be `!process.env.ELECTRON_WEBPACK_DISABLE_SOURCE_MAP_SUPPORT` for retaining the default functionality? Now if the "ELECTRON_WEBPACK_DISABLE_SOURCE_MAP_SUPPORT" variable is set to any non-empty value it adds the source-map-support which is counter-intuitive as variable is to "DISABLE".

Reply:
Wow, I can't believe I didn't spot this before pushing :man_facepalming: 

```suggestion
  if (!process.env.ELECTRON_WEBPACK_DISABLE_SOURCE_MAP_SUPPORT) {
```